cleverdiff/cleverdiff.py

```diff
@@ -85,14 +85,37 @@ def main_diff(filepairs, context_cls=None):
     print(summarise_results(diffseen, diffresult))
 
 
-def main(in_file_pairs=None):
-    if in_file_pairs is None:
-        in_file_pairs = sys.argv[1:]
-    file_pairs = [item.split("=") for item in in_file_pairs]
-
-    if not in_file_pairs:
-        raise ValueError("insufficient arguments")
-
+def main():
+    parser = argparse.ArgumentParser(
+        prog="cleverdiff",
+        description="A tool for describing multiple differences in files",
+    )
+    parser.add_argument(
+        "--old",
+        dest="old_files",
+        metavar="file",
+        type=str,
+        nargs="+",
+        help="list of reference files",
+    )
+    parser.add_argument(
+        "--new",
+        dest="new_files",
+        metavar="file",
+        type=str,
+        nargs="+",
+        help="list of modified files",
+    )
+
+    args = parser.parse_args()
+    if args.old_files is None or args.new_files is None:
+        raise ValueError("No files specified")
```

Comment:
This exception is raised if neither `--old` nor `--new` is provided, but also if the user forgets one of them. For the latter case, could you provide a specific error for when one of them is specified but not the other (e.g. "both --old and --new arguments must be specified")?

Reply:
Would still like a specific error to be shown for when `--old` or `--new` is specified but not both (e.g. "both --old and --new arguments must be specified").

Reply:
The exception is added